frontend/src/pages/Login/reducer.js

```diff
@@ -82,11 +84,28 @@ export default function loginReducer(state = defaultState, action) {
     case STORE_PASSWORD:
       return state.set("password", action.password);
     case FETCH_USER_SUCCESS:
-      const userMap = {};
+      const userDisplayNameMap = {};
+      const enabledUsers = [];
+      const disabledUsers = [];
+      const groupList = [];
       action.user.forEach(user => {
-        userMap[user.id] = user.displayName;
+        userDisplayNameMap[user.id] = user.displayName;
+        if (!user.isGroup) {
+          user.permissions["user.authenticate"].includes(user.id) ? enabledUsers.push(user) : disabledUsers.push(user);
+        } else {
+          groupList.push(user);
+        }
+      });
+      return state.merge({
+        // All users: e.g. needed in histories (MAY BE NOT NEEDED ???)
+        user: fromJS(action.user),
+        groupList: fromJS(groupList),
+        userDisplayNameMap: fromJS(userDisplayNameMap),
+        // Only enabled users: e.g. needed for assignee-lists
```

Comment:
var names are good enough i suggest to delete the comments

Reply:
done